docker-compose.yml

```diff
@@ -1,84 +1,45 @@
 version: '2'
-
 services:
-    pretalx:
-        image: pretalx
-        hostname: pretalx_core
-        container_name: pretalx_core
-        build:
-          context: .
-        volumes:
-           - ./conf/pretalx.cfg:/etc/pretalx.cfg:ro
-           - ./volumes/pretalx-data:/data:rw
-           - /src/static.dist
-        links:
-           - db:db
-           - redis:redis
-           - smtp:smtp
-        depends_on:
-           - db
+  pretalx:
+    image: maxrink/pretalx-devel:latest
+    container_name: pretalx
+    restart: unless-stopped
+    depends_on: 
+      - redis
+      - db
+    ports:
+      - "80:80"
+    volumes: 
+      - ./conf/pretalx.cfg:/etc/pretalx/pretalx.cfg:ro
+      - pretalx-data:/data
+    labels:
+      - traefik.docker.network: "pretalxdocker"
```

Comment:
@MaxRink this labels section, at least on my tests, need to be strings, such as:

`    labels:
          traefik.docker.network: "pretalxdocker"
          traefik.enable: "true"`

otherwise docker-compose throws the following error:
`ERROR: The Compose file './docker-compose.yml' is invalid because:
services.pretalx.labels contains {"traefik.docker.network": "pretalxdocker"}, which is an invalid type, it should be a string`
>

Reply:
Other that this, looks good!